### DISCUSSION TITLE
CDN LINK

### DISCUSSION THREAD

**Discussion by mikii123456789** [mdbootstrap/TW-Elements#2337] [Support from community]
I am using CDN however after copying e.g. offcanvas nothing happens.

**Comment by mikii123456789** [comment]
The modules on the site already contain "TWE", while the CDN link does not support it.

**Comment by iprzybysz** [comment]
Hi @mikii123456789, try out this CDN link: https://cdn.jsdelivr.net/npm/tw-elements/js/tw-elements.umd.min.js . Now it should work correctly.
